docs/make.jl

```diff
@@ -11,6 +12,15 @@ DocMeta.setdocmeta!(
     MCMCDiagnosticTools, :DocTestSetup, :(using MCMCDiagnosticTools); recursive=true
 )
 
+links = InterLinks(
+    "MLJ" => "https://juliaai.github.io/MLJ.jl/stable/",
+    "Statistics" => "https://docs.julialang.org/en/v1/",
+    "StatsBase" => (
+        "https://juliastats.org/StatsBase.jl/stable/",
+        "https://juliastats.org/StatsBase.jl/dev/objects.inv",
```

Comment:
Can this be removed once there is a stable release of StatsBase with docs built with Documenter >= 1?

Reply:
Yes it can. In the meantime, it's harmless, because DocumenterInterLinks will use the first valid inventory, so as soon as the stable one is available, it'll point to that.